firefox-ios/firefox-ios-tests/Tests/XCUITests/NewTabSettings.swift

```diff
@@ -134,6 +138,7 @@ class NewTabSettingsTest: FeatureFlaggedTestBase {
         mozWaitForElementToExist(app.collectionViews.links[AccessibilityIdentifiers.FirefoxHomepage.TopSites.itemCell])
     }
 
+    // https://mozilla.testrail.io/index.php?/cases/view/2307028
     func testChangeNewTabSettingsShowFirefoxHome_tabTrayExperimentOff_swipingTabsExperimentOn() {
```

Comment:
I forgot how we could have multiple experiments affecting the same test case. :(